cookbook/integrations/flytekit_plugins/modin_examples/applymap_function.py

```diff
@@ -0,0 +1,80 @@
+"""
+``applymap`` Function
+----------------------------
+
+In this example, let's manipulate the Modin DataFrame using the ``applymap`` function.
+
+"""
+
+# %%
+# Import ``ray`` and close the previous instances (if any) and start a new instance.
+import ray
+
+ray.shutdown()  # close previous instance of ray
+ray.init()  # open a new instance of ray
+
+import time
+import typing
+from types import ModuleType
+from typing import Tuple
+
+import flytekitplugins.modin
+
+# %%
+# Next, import the necessary dependencies.
+import modin.pandas
+import numpy as np
+import pandas as pd
+from flytekit import task, workflow
+
+
+# %%
+# Data Generation
+# ================
+#
+# Let's define a task to generate data (only once) using the ``random`` method. This data is stored in a Modin Dataframe.
+@task
+def generate_data() -> modin.pandas.DataFrame:
+    df = modin.pandas.DataFrame(
+        data=np.random.randint(999, 999999, size=(300000, 4)),
+        columns=[f"C{i + 1}" for i in range(4)],
+    )
+    return df
+
+
+# %%
+# Invoke the ``applymap`` Method
```

Comment:
You haven't removed the headings? We discussed about this, right? They are too near to each other. If you have at least 2 tasks beneath every heading, it makes sense, or else, it looks clumsy.

Reply:
Yes we discussed it. I have put the two tasks (`calculate_time_pandas` and `calculate_time_modin`) under a single heading. Now, I will add three tasks together- `calculate_time_pandas`, `calculate_time_modin` and `compare_time`.